util/cron/nightly_email.pl

```diff
@@ -167,3 +141,8 @@
         system("cp -pv $sortedsummary $prevsummary");
     }
 }
+
+return(0);
+
+}
```

Comment:
Does this resolve this error:   Unmatched right curly bracket at /ptmp/jenkins/chapel-ci/workspace/test-email/util/cron/nightly_email.pl line 147, at end of line
syntax error at /ptmp/jenkins/chapel-ci/workspace/test-email/util/cron/nightly_email.pl line 147, near "}" in this run:  https://chapel-ci.us.cray.com/job/test-email/82/console

Reply:
Removed in the latest patch